content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -6,6 +6,13 @@ weight: 100
 
 {{<glossary_definition term_id="node-pressure-eviction" length="short">}}</br>
 
+{{<note>}}
+The evictions signals, thresholds and other metrics that are introduced as part of
+the "Split Image" filesystem feature, which enables support for the `containerfs`
+filesystem, currently require the `KubeletSeparateDiskGC` feature gate to be enabled,
+see [KEP-4191](https://kep.k8s.io/4191) for more details.
```

Comment:
Good to know!

I have since removed the reference to the KEP. Especially, since we also include details about different types of filesystems Kubernetes supports and the `containersfs` itself within the chapters of this document.

Hopefully, this is sufficient.